### PR TITLE
Added commands to set/unset birthday

### PR DESCRIPTION
**Pull request by ErisApps**
Basically another extension on the UserSettings feature that was introduced last time
For now, this doesn't really do much aside from updating the birthday date in the database, but will be used to power a birthday Quartz job in the near future 😸

### CODE REVIEW

File: POI.DiscordDotNet/Commands/Profile/BirthdaySlashCommandsModule.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			[SlashCommand("set", "Sets your birthday")]
+			public async Task Set(InteractionContext ctx, [Option("date", "The date of your birthday. (Format: yyyy-MM-dd)")] string birthdayDateRaw)
+			{
+				var parseResult = LocalDatePattern.Iso.Parse(birthdayDateRaw);
+				if (parseResult.Success)
+				{
+					await _userSettingsService.UpdateBirthday(ctx.User.Id.ToString(), parseResult.Value).ConfigureAwait(false);
+				}
+
+				await ctx.CreateResponseAsync("Birthday has been updated").ConfigureAwait(false);
```
Comment (grYs51) [inline POI.DiscordDotNet/Commands/Profile/BirthdaySlashCommandsModule.cs:29]: Always responding with b-day updated even though it didn't

File: POI.DiscordDotNet/Services/SlashCommandsManagementService.cs [hunk 2]
```diff
@@ -32,6 +33,8 @@ public void Setup()
 			// TODO: Register slash commands below
 			_slashCommands.RegisterCommands<PingCommand>();
 			_slashCommands.RegisterCommands<UptimeCommand>();
+
```
Comment (grYs51) [inline POI.DiscordDotNet/Services/SlashCommandsManagementService.cs:36]: ?

**Review by grYs51 — APPROVED** [review]

File: POI.DiscordDotNet/Services/SlashCommandsManagementService.cs (hunk 2 above)
Comment (ErisApps) [reply POI.DiscordDotNet/Services/SlashCommandsManagementService.cs:36]: Intentional, Ping and Uptime are both part of the Utils module while Birthday commands are part of the profile commands module ^^

File: POI.DiscordDotNet/Commands/Profile/BirthdaySlashCommandsModule.cs (hunk 1 above)
Comment (ErisApps) [reply POI.DiscordDotNet/Commands/Profile/BirthdaySlashCommandsModule.cs:29]: Good catch, will feex

File: POI.DiscordDotNet/Commands/Profile/BirthdaySlashCommandsModule.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
+﻿using System.Threading.Tasks;
+using DSharpPlus.SlashCommands;
+using NodaTime.Text;
+using POI.DiscordDotNet.Services;
+
+namespace POI.DiscordDotNet.Commands.Profile
+{
+	public partial class ProfileSlashCommandsModule
+	{
+		[SlashCommandGroup("birthday", "Commands related to your birthday")]
```
Comment (GianniKoch) [inline POI.DiscordDotNet/Commands/Profile/BirthdaySlashCommandsModule.cs:10]: Minor thing but what about "Commands related to managing your birthday"?

**Review by GianniKoch — APPROVED** [review]

**Pull request merged**